frontend/src/components/navigation/menuItem.tsx

```diff
@@ -1,44 +1,44 @@
-import {Menu as AMenuItem} from "antd";
-import styled from "styled-components";
-import {ProfileFilled} from '@ant-design/icons';
+import { Menu } from 'antd';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 
+const Container = styled.div`
```

Comment:
Na pewno trzeba to trochę ładniej poukładać, dodaj tabulację to poniższych styli (wkrótce będzie prettier, ale on tego nie wyłapie tutaj)